src/pages/Attendance.jsx

```diff
@@ -13,43 +13,138 @@ import ChurchIcon from "@mui/icons-material/Church"
 import ArrowLeftIcon from "@mui/icons-material/ArrowLeft"
 import ArrowRightIcon from "@mui/icons-material/ArrowRight"
 import SettingsIcon from "@mui/icons-material/Settings"
+import AddBoxOutlinedIcon from "@mui/icons-material/AddBoxOutlined"
+import IndeterminateCheckBoxOutlinedIcon from "@mui/icons-material/IndeterminateCheckBoxOutlined"
 
 import Layout from "../components/Layout/Layout"
 import Title from "../components/shared/Title"
 import { mcyMember } from "../data/mcyMember"
 
 const Attendance = () => {
-  const [value, setValue] = React.useState(dayjs())
   const [isOpenCalendar, setIsOpenCalendar] = React.useState(false)
   const [selectedLeader, setSelectedLeader] = React.useState(null)
-  const [isChecked, setIsChecked] = React.useState(false)
+  const [adultCount, setAdultCount] = React.useState(0)
+  const [memberCount, setMemberCount] = React.useState(0)
+  const totalCount = adultCount + memberCount
+  const [attendanceAllData, setAttendanceAllData] = React.useState({})
+  const [isChecked, setIsChecked] = React.useState({})
+  const currentDayOfWeek = dayjs().day()
+  const daysToLastSunday = currentDayOfWeek === 0 ? 7 : currentDayOfWeek
+  const lastSunday = dayjs().subtract(daysToLastSunday, "day")
+  const [value, setValue] = React.useState(dayjs(lastSunday))
+
+  React.useEffect(() => {
+    const initialCheckedState = {}
+    mcyMember.forEach(leader => {
+      leader.cellMember.forEach(member => {
+        initialCheckedState[member] = false
+      })
+    })
+    setIsChecked(initialCheckedState)
+    setSelectedLeader(null)
+    setAdultCount(0)
+    setMemberCount(0)
+    setAttendanceAllData({})
```

Comment:
```suggestion
    setIsChecked(initialCheckedState)
    setSelectedLeader(null)
    setAdultCount(0)
    setMemberCount(0)
    setAttendanceAllData({})
```
useState를 값이 아니라 객체로 관리하는 방법도 있음. 아래처럼. 어떤 방법이 나을지 객체로 관리하는 것도 한번 적용해보면서 비교해보면 좋을듯
```
useState({
 adultCount:0,
 memberCount:0,
...
})
```